src/regression_tracker.py

```diff
@@ -30,44 +31,71 @@ def _stop(self, sub_id):
         if sub_id:
             self._api_helper.unsubscribe_filters(sub_id)
 
-    def _create_regression(self, failed_node, last_successful_node):
+    def _create_regression(self, failed_node, last_pass_node):
         """Method to create a regression"""
         regression = {}
-        for field in self._regression_fields:
-            regression[field] = failed_node[field]
-
         regression['kind'] = 'regression'
+        regression['name'] = failed_node['name']
+        regression['path'] = failed_node['path']
+        regression['group'] = failed_node['group']
+        regression['state'] = 'done'
```

Comment:
I had added  `self._regression_fields` for easy field assignment. May I know the reason for dropping it?

Reply:
Yes, that's why suggested having a different list for data fields.
I'd do something like this:
```
        for field in self._regression_data_fields:
            regression['data'][field] = failed_node['data'].get(field)
```
IMO that would reduce lines of code and it's also easy while adding more data fields. This would just require adding the field to the list.
If we decide to replace this with the helper function, we can use this same logic in that function as well.

Reply:
Oh okay. So it's already been merged.
Got it. Sorry for the noise.